src/Entities/Player.h

```diff
@@ -413,7 +414,16 @@ class cPlayer :
 	If the player is not riding a horse or if the horse is untamed, does nothing. */
 	void OpenHorseInventory();
 
-	void UseEquippedItem(int a_Amount = 1);
+	/** Damage the player's equipped item by a_Damage, possibly less if the
+	equipped item is enchanted. */
+	void UseEquippedItem(short a_Damage);
+
+	/** Shorthand for UseEquippedItem(1) */
+	void UseEquippedItem(void);
```

Comment:
What was wrong with using a default value?

Reply:
That's a much better idea, I didn't think about the possibility.